src/main/java/uk/gov/hmcts/ccd/domain/service/accessprofile/filter/matcher/AttributeMatcher.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ccd.domain.service.accessprofile.filter.matcher;
+
+import java.util.Optional;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignmentFilteringResult;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+public interface AttributeMatcher {
+
+    boolean matchAttribute(RoleAssignmentFilteringResult result, CaseDetails caseDetails);
```

Comment:
any particular reason for a boolean return type? it's not used

Reply:
removed boolean return